lib/citizens_advice_form_builder/elements.rb

```diff
@@ -6,6 +6,7 @@
 require_relative "elements/text_area"
 require_relative "elements/date_input"
 require_relative "elements/error_summary"
+require_relative "elements/check_box"
```

Comment:
You should `require` the design system component in `lib/citizens_advice_form_builder/form_builder.rb` too.